nilearn/plotting/displays.py

```diff
@@ -746,12 +746,10 @@ def add_edges(self, img, color='r'):
 
         Parameters
         ----------
-        map: 3D ndarray
-            The 3D map to be plotted. If it is a masked array, only
-            the non-masked part will be plotted.
-        affine: 4x4 ndarray
-            The affine matrix giving the transformation from voxel
-            indices to world space.
+        img: Niimg-like object
+            See http://nilearn.github.io/manipulating_images/input_output.html.
+            The 3D map to be plotted.
+            If it is a masked array, only the non-masked part will be plotted.
```

Comment:
Sorry for not being clear. I expected something like this,

"The 3D image to be plotted. If it is masked, only the non-masked part will be plotted/projected."